app/script_variant.py

```diff
@@ -43,31 +49,39 @@ def romanize_japanese_name_by_name_dict(word):
     This method romanizes japanese name by using name dictionary.
     If word isn't found in dictionary, this method doesn't
     apply romanize.
+    This method can return multiple romanizations.
+    Returns:
+        [romanized_jp_name, ...]
     """
     if not word:
-        return word
+        return ['']
 
     if word in JAPANESE_NAME_DICTIONARY:
-        yomigana = JAPANESE_NAME_DICTIONARY[word]
-        return jautils.hiragana_to_romaji(yomigana)
+        yomigana_list = JAPANESE_NAME_DICTIONARY[word]
+        return [jautils.hiragana_to_romaji(yomigana)
+                for yomigana in yomigana_list]
 
-    return word
+    return [word]
 
 
 def romanize_japanese_location(word):
     """
-    This method romanizes japanese name by using name dictionary.
+    This method romanizes japanese location by using name dictionary.
     If word isn't found in dictionary, this method doesn't
     apply romanize.
+    This method can return multiple romanizations.
```

Comment:
Non-Japanese people may not know that there are multiple ways to read the same kanji name. Please explain it here.